WooCommerce/Classes/ViewRelated/CardPresentPayments/CardPresentPaymentsModalViewController.swift

```diff
@@ -158,7 +158,7 @@ private extension CardPresentPaymentsModalViewController {
     }
 
     func styleAuxiliaryButton() {
-        if viewModel.actionsMode != .secondaryActionAndAttributedAuxiliaryButton {
+        if viewModel.actionsMode != .secondaryActionAndAuxiliaryButton {
```

Comment:
I didn't notice this one before but again it feels like a misuse of the `actionsMode`. Another view may in future want a secondary action and a link-styled auxiliary button.

Perhaps a better property to check would be `viewModel.attributedAuxiliaryButtonTitle` because it seems fairly safe to assume that we don't want to re-style an attributed string as a link?